recursio-client/Characters/Visuals/Particles/ParticleController.gd

```diff
@@ -0,0 +1,17 @@
+extends Particles
+
+export var time_until_cleanup = 1.0
+
+export var _time_existing = 0.0
+
+func start():
+	restart()
+	_time_existing = 0.0
+
+func stop():
+	emitting = false
+
+func _process(delta):
+	_time_existing += delta
+	if time_until_cleanup >=0 and _time_existing >= time_until_cleanup:
+		get_parent().remove_child(self)
```

Comment:
Do you still need it or why are you just removing it from the tree instead of using `queue_free` here?

Reply:
Oh I'm not even using that script anymore, just forgot to delete it. Should be gone with b8133a8.